test/e2e/wait_test.go

```diff
@@ -64,11 +64,26 @@ var _ = Describe("Podman wait", func() {
 		session = podmanTest.Podman([]string{"wait", "-l"})
 		session.Wait(20)
 	})
+
 	It("podman container wait on latest container", func() {
 		session := podmanTest.Podman([]string{"container", "run", "-d", ALPINE, "sleep", "1"})
 		session.Wait(20)
 		Expect(session.ExitCode()).To(Equal(0))
 		session = podmanTest.Podman([]string{"container", "wait", "-l"})
 		session.Wait(20)
 	})
+
+	It("podman wait on two containers", func() {
+		session := podmanTest.Podman([]string{"run", "-d", ALPINE, "sleep", "1"})
+		session.Wait(20)
+		cid1 := session.OutputToString()
+		session = podmanTest.Podman([]string{"run", "-d", ALPINE, "sleep", "1"})
+		session.Wait(20)
+		cid2 := session.OutputToString()
+		Expect(session.ExitCode()).To(Equal(0))
+		session = podmanTest.Podman([]string{"wait", cid1, cid2})
+		session.Wait(20)
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(session.OutputToStringArray()).To(Equal([]string{"0", "0"}))
+	})
```

Comment:
This test is fine, but 3 tends to be a fun number.  Could you create a test with 3 images put into wait?

Reply:
Done